files/nginx-mc2.ctmpl

```diff
@@ -0,0 +1,23 @@
+# NOTE: This is an nginx configuration automatically generated by
+#       consul-template, there's no point in making manual changes
+#       to this file.
+#
+#       This expects a key to exist under "consular/<app_id>/domain"
+#       in Consul's KV store. This is used to map a domain to a service in
+#       Nginx. If the key does not exist the service isn't added to the
+#       list of services in the Nginx config.
+
+{{range services}}{{$labels := ls (print "consular/" .Name) | explode }}{{if eq .Name "seed-infra-mc2-mc2"}}
+server {
+  listen 7001;
```

Comment:
Nope. I needed to expose it somewhere so I could test that it was working, and the docker run command I used for reference put it on 7001.